lib/statsd/instrument/prometheus/prometheus_sink.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module StatsD
+  module Instrument
+    module Prometheus
+      class PrometheusSink
+        THREAD_NAME = "StatsD::PrometheusSink"
```

Comment:
I'm still going commit by commit, so maybe there will be changes later, but for now is the only change between this and `UDPSink` just the thread name?
If so, do we want to parameterise that instead of having a new duplicated class?

Reply:
nvm, got to the changes now.

Reply:
and got to the bit where we're sharing again. Perfect! https://github.com/meetcleo/statsd-instrument/pull/1/commits/9f6e819e32a8204cfcf33715aec39fb924c1a2f4 :ok_hand: